docs/run_test/run.md

```diff
@@ -0,0 +1,23 @@
+# Run LISA
+
+- ## [Quick start](quick_run.md)
+  - ### [Getting started with Azure](quick_run.md)
+- ## [Learn Microsoft tests](microsoft_tests.md)
+  - ### [How to run](microsoft_tests.md#how-to)
+  - ### [Test priority](microsoft_tests.md#test-priority)
+  - ### [Test tier](microsoft_tests.md#test-tier)
+  - ### [Test cases specification](microsoft_tests.md#test-cases-specification)
+  - ### [LISAv2 legacy tests](run_legacy.md)
```

Comment:
Let's call it `legacy test`, no "v2". The v2 is confusing, because we don't call current version is v3 externally.